src/web/src/main/java/tupperdate/web/model/Recipe.kt

```diff
@@ -6,16 +6,18 @@ import tupperdate.common.dto.RecipeDTO
 
 data class Recipe(
     val id: String? = null,
+    val userId: String? = null,
     val title: String? = null,
     val description: String? = null,
     val timestamp: Long? = null,
     val picture: String? = null,
     val attributes: Map<String, Boolean>? = null,
 )
 
-fun NewRecipeDTO.toRecipe(id: String, picture: String): Recipe {
+fun NewRecipeDTO.toRecipe(id: String, userId: String, picture: String): Recipe {
     return Recipe(
         id = id,
+        userId = userId,
         title = this.title,
         description = this.description,
         timestamp = System.currentTimeMillis() / 1000,
```

Comment:
This is also something we'll want to control.

Reply:
This will be addressed in the next iteration using transactions